src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -1465,6 +1467,117 @@ class Simplex_tree {
     }
   }
 
+  /** \brief Retrieve good values for extended persistence, and separate the 
```

Comment:
"good" values are only approximate with this implementation (numerical errors when going to [-2,-1] and back), I think we want to state that clearly.

Reply:
Done.